clients/unshaded/src/main/java/tachyon/client/file/options/CreateFileOptions.java

```diff
@@ -178,9 +178,8 @@ public String toString() {
   /**
    * @return Thrift representation of the options
    */
-  // TODO(calvin): Rename CreateTOptions to CreateFileTOptions
-  public CreateTOptions toThrift() {
-    CreateTOptions options = new CreateTOptions();
+  public CreateFileTOptions toThrift() {
```

Comment:
Not sure I understand what you said. I think there will be two counterparts: an option exposed to users and the other for thrift. The interface will have to/from method for converting between them.